docs/user-guide/api-mediation/configuration-unique-cookie-name-for-multiple-zowe-instances.md

```diff
@@ -8,11 +8,14 @@ To prevent overwriting of the default cookie name in the case of multiple Zowe i
 
 Follow this procedure to configure a unique cookie name for the instances:
 
-1. Open the `zowe.yaml` configuration file.
-2. Find or add the property  `components.gateway.apiml.security.auth.uniqueCookie`, and set it to `true`. A unique cookie name is generated as `apimlAuthenticationToken.cookieIdentifier`.
+1. Open the Zowe YAML configuration file.
```

Comment:
For the live site and previous versions:
- Why are we not saying `zowe.yaml` like we did before? 
- I think we should say how to open it, like: "Use a text editor to open the `zowe.yaml` configuration file."

Reply:
Because zowe.yaml is incorrect since the YAML may not even be a file with a .yaml suffix

https://docs.zowe.org/stable/user-guide/configmgr-using#splitting-configuration-into-multiple-storage-types documents that YAML can be stored in PARMLIB
Such parmlibs wouldnt be named thing.yaml, because they aren't files in the traditional sense.
For lack of a better way of saying it, in docs I tend to say "Zowe YAML configuration file"